charts/k8ssandra/templates/_helpers.tpl

```diff
@@ -117,24 +117,41 @@ Return the ingress host that should be used for Stargate's Cassandra/CQL interfa
 
 {{/*
 Create the jvm options based on heap properties specified.
+Expecting that c* heap.size and heap.newGenSize are NOT in IEC format.
 */}}
 {{- define "k8ssandra.configureJvmHeap" -}}
 {{- $datacenter := (index .Values.cassandra.datacenters 0) -}}
 {{- if $datacenter.heap }}
   {{- if $datacenter.heap.size }}
-      initial_heap_size: {{ $datacenter.heap.size }}
-      max_heap_size: {{ $datacenter.heap.size }}
+      {{- if (regexMatch "^(([0]\\.\\d*)+|(^[1-9]\\d*)+(\\.\\d+)?)(?i)(k|m|g|e|p|t){1}$" (print $datacenter.heap.size) ) }}
+        {{- nindent 8 (print "initial_heap_size: " $datacenter.heap.size) }}
```

Comment:
Indentation is off by 2 and the same for the other `nindent` calls.

Reply:
Good catch!

To assist with any future changes in `_helpers.tpl`, I've included a test utility in the upcoming push.  It provides support for an optional flag to capture the rendered output in a file.  The option takes a destination `dir` and `name` of the rendered output file.

```json
--rendertmpl={ "dir":"/tmp/foobar","name":"cluster_heap_size_valid.yaml"}
```

Usage:
```go
    options := &helm.Options{
      SetValues: map[string]string{
        "cassandra.heap.size":           "4.5G",
	"cassandra.heap.newGenSize":     "1.5G",
	"cassandra.datacenters[0].name": dcName,
	helmUtils.OPT_RENDER_TEMPLATE:   `{ "dir":"/tmp/foobar","name":"cluster_heap_size_valid.yaml"}`,
    },
    KubectlOptions: defaultKubeCtlOptions,
}
```